root/static/scripts/edit/URLCleanup.js

```diff
@@ -1337,6 +1337,80 @@ const CLEANUPS: CleanupEntries = {
       };
     },
   },
+  'boomkat': {
+    match: [new RegExp('^(https?://)?(www\\.)?boomkat\\.com', 'i')],
+    restrict: [
+      {
+        artist: [
+          LINK_TYPES.downloadpurchase.artist,
+          LINK_TYPES.mailorder.artist,
+        ],
+        label: [
+          LINK_TYPES.downloadpurchase.label,
+          LINK_TYPES.mailorder.label,
+        ]
```

Comment:
This is missing a dangling comma and triggering a linting error :)

Reply:
Same but actionable suggestion:
```suggestion
        ],
```